media/gpu/v4l2/v4l2_device.cc

```diff
@@ -93,6 +93,19 @@ uint32_t V4L2Device::VideoPixelFormatToV4L2PixFmt(VideoPixelFormat format) {
 }
 
 // static
+#if BUILDFLAG(USE_LINUX_V4L2)
+uint32_t V4L2Device::VideoCodecProfileToV4L2PixFmt(VideoCodecProfile profile,
```

Comment:
can we do something like this?

````
uint32_t V4L2Device::VideoCodecProfileToV4L2PixFmt(VideoCodecProfile profile,
                                                                                          bool slide_based) {
#if defined(USE_LINUX_V4L2)
  // On linux we force slide_based to false, so that the code flow goes as expected below.
  slide_based = false;
#endif
  <keep the code as it originally is in the method>
}

Reply:
Hi Antonio,
I think that would be nice if the "*_FRAME" and "*_SLICE" are defined.  These are not defined in mainline Linux headers. When I made the original patch, they weren't defined in Chromium either, only the modified V4L2 headers for ChromeOS.  That might have changed recently.